src/Vegetation/Roots.jl

```diff
@@ -108,26 +108,18 @@ abstract type AbstractTranspiration{FT <: AbstractFloat} end
 
 """
     RootsParameters{FT <: AbstractFloat}
-A struct for holding parameters of the Root Model.
+A struct for holding parameters of the Root Model. Eventually to be used with ClimaParameters.
 $(DocStringExtensions.FIELDS)
 """
 struct RootsParameters{FT <: AbstractFloat, PSE}
-    "controls the shape and steepness of conductance vs. pressure curve, for roots: unitless"
-    a_root::FT
-    "controls the steepness of the relative conductance vs. pressure curve, for roots: inverse MPa"
-    b_root::FT
-    "controls the shape and steepness of relative conductance vs. pressure curve, for stems: unitless"
-    a_stem::FT
-    "controls the steepness of the conductance vs. pressure curve, for stems: inverse MPa"
-    b_stem::FT
-    "the physical size of the stem, in moles"
-    size_reservoir_stem_moles::FT
-    "the physical size of the leaves, in moles"
-    size_reservoir_leaf_moles::FT
-    "water conductance in roots (moles/s/MPa) when pressure is zero, a maximum"
-    K_max_root_moles::FT
-    "water conductance in stems (moles/s/MPa) when pressure is zero, a maximum"
-    K_max_stem_moles::FT
+    "controls the shape and steepness of conductance vs. pressure curve, for roots, stem compartments, and leaf compartments: unitless"
+    a_variables::NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}}
```

Comment:
We should rename these (`a`, `b`), they are not informative... But @gagnelandmanna can let us know if these will be renamed (or removed) in her PR.